src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -104,7 +104,7 @@ const AdminDashboard = () => {
   useEffect(() => {
     axios
       .get(
-        'https://hrf-bw-labs37-dev.eba-hz3uh94j.us-east-1.elasticbeanstalk.com/to-approve'
+        'http://hrf-bw-labs37-dev.eba-hz3uh94j.us-east-1.elasticbeanstalk.com/to-approve'
```

Comment:
After trying both addresses it seems the AWS is currently only set up for http use. At this time, this would probably be the bug fix to this problem.  However it may be advisable to look into AWS to see how we could set up https.

Reply:
Seconding this opinion - AWS is currently configured for HTTP use, but in the future, it may be worthwhile to look into configuring for HTTPS for added security.